Assets/MRTK/SDK/Features/Utilities/Solvers/TapToPlace.cs

```diff
@@ -306,6 +345,13 @@ protected virtual void SetPosition()
             {
                 // Take the current hit point and add an offset relative to the surface to avoid half of the object in the surface
                 GoalPosition = CurrentHit.point;
+
+                // Allow switching between a specified SurfaceNormalOffset and the defaultSurfaceNormalOffset while the object is in the placing state
+                if (UseDefaultSurfaceNormalOffset && SurfaceNormalOffset != defaultSurfaceNormalOffset)
+                {
+                    SurfaceNormalOffset = defaultSurfaceNormalOffset;
```

Comment:
This is the difference between caching and not caching the defaultSurfaceNormalOffset. I think the scenario is not the most likely, but I think it's nice to have the auto adjustment if the size changes while placing.

| Without Bounds Update | With Bounds Update |
|----|----|
| ![TapToPlaceBoundsNoUpdate](https://user-images.githubusercontent.com/53493796/84827088-cc15d100-afd8-11ea-8560-b221c384bb9c.gif)| ![TapToPlaceBoundsUpdate](https://user-images.githubusercontent.com/53493796/84827123-d637cf80-afd8-11ea-9347-455733b62b0a.gif) |

Reply:
In that video you're resizing the box with the mouse while at the same time "placing" it (though the "placing" part isn't shown since it's not being moved in any way). I personally don't feel that would happen very often, but calculating the bounds of a collider is not exceedingly costly so I'm OK if you feel strongly about keeping it.